service/frontend/workflow_handler.go

```diff
@@ -5188,3 +5188,42 @@ func (wh *WorkflowHandler) UpdateActivityOptionsById(
 		ActivityOptions: response.ActivityOptions,
 	}, nil
 }
+
+func (wh *WorkflowHandler) PauseActivityById(
+	ctx context.Context,
+	request *workflowservice.PauseActivityByIdRequest,
+) (_ *workflowservice.PauseActivityByIdResponse, retError error) {
+	defer log.CapturePanic(wh.logger, &retError)
+
+	if !wh.config.ActivityAPIsEnabled(request.GetNamespace()) {
+		return nil, status.Errorf(codes.Unimplemented, "method PauseActivity not implemented")
+	}
+
+	wh.logger.Debug("Received PauseActivity")
```

Comment:
We should delete this log before merging IMHO.

Reply:
removed